external-api/src/interfaces/eStore.interface.ts

```diff
@@ -182,21 +195,25 @@ export interface DealFolderResponse {
 
 /**
  * Represents the response for a facility folder operation within the eStore.
+ *
  * @interface FacilityFolderResponse
  * @extends EstoreResponse
  * @property {string} data.foldername - Name of the facility folder.
  * @property {string} data.error - Error message, if any.
+ * @property {unknown} data.message - Additional message, if any.
  */
 export interface FacilityFolderResponse {
   readonly status: number;
   readonly data: {
     foldername?: string;
     error?: string;
+    message?: unknown;
```

Comment:
Why is this unknown, could the response be anything or maybe 1 of a 2/3 possible responses?